core/src/apps/wallet/sign_tx/addresses.py

```diff
@@ -214,21 +214,29 @@ def validate_full_path(
     ):
         return False
 
-    if path[1] != coin.slip44 | HARDENED:
+    if path[1] > 20 and path[1] != coin.slip44 | HARDENED:
```

Comment:
Why `path[1] > 20`?

Reply:
The main reason is that BIP45 has the following format:

```
m / purpose' / cosigner_index / change / address_index
```

So `path[1]` can be also non-hardened